snapshots/testsuite/testsuite.go

```diff
@@ -891,19 +902,19 @@ func check128LayersMount(name string) func(ctx context.Context, t *testing.T, sn
 		lowestApply := fstest.Apply(
 			fstest.CreateFile("/bottom", []byte("way at the bottom\n"), 0777),
 			fstest.CreateFile("/overwriteme", []byte("FIRST!\n"), 0777),
-			fstest.CreateDir("/ADDHERE", 0755),
-			fstest.CreateDir("/ONLYME", 0755),
-			fstest.CreateFile("/ONLYME/bottom", []byte("bye!\n"), 0777),
+			fstest.CreateDir("/addhere", 0755),
```

Comment:
Do you need to change them?

Reply:
Yes. Windows has the amusing behaviour that an all-caps file is assumed to be a DOS legacy, and recased before writing. So it's either change the case, or make the comparison code case-insensitive. See https://github.com/containerd/containerd/pull/8043/commits/a9bf2f30dcf4eec768864179b82785cbad216355

Reply:
> Yes. Windows has the amusing behaviour that an all-caps file is assumed to be a DOS legacy, and recased before writing. So it's either change the case, or make the comparison code case-insensitive. See [a9bf2f3](https://github.com/containerd/containerd/commit/a9bf2f30dcf4eec768864179b82785cbad216355).

The mine fields we tend to create when we can't stop ourselves from adding code that tries to be smarter than it needs to be. Thanks for the context! TIL :smile: